src/ethereum/curve/contracts.ts

```diff
@@ -4881,7 +4881,7 @@ const contracts = {
       ],
     },
   },
-  CurveMetapoolSwapPool: {
+  CurveMetapoolSwapAdapter: {
```

Comment:
Small doubt: Is `Adapter` suffix is required? If so, then For other existing keys like `CurveDepositPool`, It is not same like Adapter name [here](https://github.com/Opty-Fi/team-defi-adapters/blob/master/contracts/1_ethereum/curve/CurveDepositPoolAdapter.sol#L38)

Reply:
Same doubt

Reply:
- [ ] adapter is not required